### PR TITLE
feat: [#1952] a/b testing for landing page

### PR DESCRIPTION
**Pull request by aloverso**

### CODE REVIEW

File: web/src/lib/types/types.ts [hunk 1]
```diff
@@ -353,7 +353,7 @@ export const owningFlowDisplayFields = Object.keys(emptyOwningFlowContent) as (k
 
 export const startFlowDisplayFields = Object.keys(emptyStartingFlowContent) as (keyof StartingFlowContent)[];
 
-const businessUserDisplayFields = Object.keys(emptyBusinessUser) as (keyof BusinessUser)[];
```
Comment (mikewolfd) [inline web/src/lib/types/types.ts:356]: I think this will mess up typing in our IDE's, no?

Comment (aloverso) [reply web/src/lib/types/types.ts:356]: Will it? I'm not sure. I don't **think** it changed anything for me?

File: web/src/lib/types/types.ts [hunk 2]
```diff
@@ -353,7 +353,7 @@ export const owningFlowDisplayFields = Object.keys(emptyOwningFlowContent) as (k
...
+const businessUserDisplayFields = Object.keys(createEmptyUser()) as (keyof BusinessUser)[];
```
Comment (mikewolfd) [inline web/src/lib/types/types.ts:356]: read below with this messing with the typing

Comment (aloverso) [reply web/src/lib/types/types.ts:356]: Is it actually causing issues? If the IDE knows the return type of the function then can't it do object.keys?

Comment (aloverso) [reply web/src/lib/types/types.ts:356]: I suppose I could override that, though... but yeah, I'm curious how this change disrupts the types?

**Pull request merged**